src/community/propose-a-content-change-using-github/index.md.njk

```diff
@@ -0,0 +1,91 @@
+---
+title: Propose a content change using GitHub
+description: Find out how to suggest a change to content in the GOV.UK Design System using GitHub
+section: Community
+layout: layout-pane.njk
+order: 2
+---
+
+The GOV.UK Design System team uses a service called GitHub to manage content in the GOV.UK Design System.
+
+This guide explains how to propose a change to the Design System's content. You'll need a GitHub account to do this.
+
+If you do not have one already, you can [create a GitHub account for free](https://github.com/).
+
+{% from "inset-text/macro.njk" import govukInsetText %}
+{{ govukInsetText({
+  text: "Rest assured that it's impossible for you to break the Design System by proposing changes. The GOV.UK Design System team reviews all changes before publishing."
+}) }}
+
+If you get stuck whilst following these steps and you need help, you can:
+
+- get in touch on [#govuk-design-system channel on cross-government Slack](https://ukgovernmentdigital.slack.com/app_redirect?channel=govuk-design-system)
+- email the GOV.UK Design System team on <br>govuk-design-system-support@digital.cabinet-office.gov.uk
+
+## 1. Go to the page you want to edit
+
+At the bottom of every page in the Design System you will find a section called 'Help improve this page'. 
+
+Follow the link to propose a change to the page. This will take you to the page's Markdown file. 
+
+You might be told you need to fork the Design System repository to make changes. This is nothing to worry about. It just means you're making a copy of the Design System that you can edit. Select "fork this repository and propose changes" to continue. 
+
+![](fork-this-repo.png)
```

Comment:
Sorry I didnt catch this earlier, we need to make sure we've got good alternative text for our images...

Reply:
Actually nevermind it's explained above.